src/Stratis.Bitcoin.Features.Consensus.Tests/TimeSyncBehaviorAdjustedTimeTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Stratis.Bitcoin.Base;
+using Stratis.Bitcoin.Utilities;
+using Xunit;
+
+namespace Stratis.Bitcoin.Features.Consensus.Tests
+{
+    /// <summary>
+    /// Tests of <see cref="TimeSyncBehavior"/> and <see cref="TimeSyncBehaviorState"/> classes.
+    /// </summary>
+    public partial class TimeSyncBehaviorAdjustedTimeTest
+    {
+        /// <summary>
+        /// Time data is added this this state. The combination of this and datetimeprovider are what is under test.
+        /// </summary>
+        private TimeSyncBehaviorState timesyncBehaviourState;
+
+        /// <summary>
+        /// This provides the adjusted time ready for assertion. The combination of this and timesyncBehaviorState are what is under test.
+        /// </summary>
+        private IDateTimeProvider dateTimeProvider;
+        
+        /// <summary>
+        /// This is the result of the operation under test. It is only rounded so that assertions are easier, a future improvement would be to not use UtcNow directly in the code, allowing it to be replaced at test time.
+        /// </summary>
+        private int roundedOffset;
+
+        /// <summary>
+        /// Checks that <see cref="TimeSyncBehaviorState.AddTimeData(IPAddress, TimeSpan, bool)"/>
```

Comment:
Those references can't be resolved by the VS so please add 
```
using System;
using System.Net;
```

Reply:
need my resharper setting for this somehow as for me it wants to remove them